internal/zinc-persist/src/main/scala/sbt/internal/inc/binary/converters/ProtobufReaders.scala

```diff
@@ -26,11 +27,17 @@ final class ProtobufReaders(mapper: ReadMapper) {
     java.nio.file.Paths.get(path).toFile
   }
 
+  private def readLong(bs: ByteString) = {
```

Comment:
See above: should just take the Array[Byte] directly here. Or better yet, use ByteBuffer in your public APIs and just directly consume `asReadOnlyByteBuffer` here.